docker-compose.yml

```diff
@@ -0,0 +1,58 @@
+version: "3.7"
+
```

Comment:
Having just done a Docker Compose conversion for my own SimpleLogin instance, the issue I ran into was with the networking. Postfix fails to send emails from the default Docker Compose network because the subnet isn't the same as the one used in the README guide / Postfix config. You could probably change the suggested Postfix config to fix this, or if you're like me and felt more comfortable messing with Docker, add the following to the compose file emulates the setup in the README and keeps everything happy as far as I can tell:

```yml
networks:
  default:
    name: sl-network
    ipam:
      config:
        - subnet: 10.0.0.0/24
```

I don't know if your use of Caddy changes or will work around this situation as I don't use it myself, but I thought it'd be worth mentioning

Reply:
Just seen you're using Postfix in a container so I'm guessing this is accounted for

Reply:
Would you mind sharing your docker compose file?

Mine doesn't work (-:

> Having just done a Docker Compose conversion for my own SimpleLogin instance, the issue I ran into was with the networking. Postfix fails to send emails from the default Docker Compose network because the subnet isn't the same as the one used in the README guide / Postfix config. You could probably change the suggested Postfix config to fix this, or if you're like me and felt more comfortable messing with Docker, add the following to the compose file emulates the setup in the README and keeps everything happy as far as I can tell:
> 
> ```yaml
> networks:
>   default:
>     name: sl-network
>     ipam:
>       config:
>         - subnet: 10.0.0.0/24
> ```
> 
> I don't know if your use of Caddy changes or will work around this situation as I don't use it myself, but I thought it'd be worth mentioning

Reply:
@Slevin90 I've posted it previously [here](https://github.com/simple-login/app/issues/916#issuecomment-1190583333). Please note the disclaimer at the bottom